src/components/common/YourCompany/BusinessStartDate.vue

```diff
@@ -155,23 +156,20 @@ export default class StartDate extends Mixins(CommonMixin, DateMixin) {
 
   /** The business date or business start date string. */
   get businessStartDate (): string {
-    // safety check
-    if (this.isApplicableFiling) {
```

Comment:
Removed this "safety check" since it prevented the display of Business Start Date in a conversion.